### PR TITLE
Feature/cors plugin

### PR DESCRIPTION
**Pull request by Nijikokun**
Here is the `cors` plugin @thibaultCha and @thefosk

### CODE REVIEW

**Comment by thefosk** [comment]
@thibaultCha any feedback on this?

File: spec/plugins/cors_spec.lua [hunk 1]
```diff
@@ -0,0 +1,220 @@
+local spec_helper = require "spec.spec_helpers"
+local http_client = require "kong.tools.http_client"
+local cjson = require "cjson"
+
+local APIS = {}
+local PLUGINS = {}
+local API_URL = spec_helper.API_URL
+local PROXY_URL = spec_helper.PROXY_URL
+
+function request (name, method, qs, headers)
```
Comment (thibaultCha) [inline spec/plugins/cors_spec.lua]: All those functions should be declared `local`. Also the `(` has no space before in all our code.

File: kong/plugins/cors/access.lua [hunk 2]
```diff
@@ -0,0 +1,68 @@
+local _M = {}
+
+local function configure_origin (ngx, conf)
```
Comment (thibaultCha) [inline kong/plugins/cors/access.lua]: Same for space before `(`

**Comment by thibaultCha** [comment]
It looks good appart for some code style but since it's a plugin, I don't know how long it will live in this repo anyways. (We should probably brace ourselves to the worst and expect them to be here for a looong time tho...)

**Comment by thibaultCha** [comment]
Also, #139

**Comment by Nijikokun** [comment]
Merging this will close #139

File: spec/plugins/cors_spec.lua (hunk 1 above)
Comment (Nijikokun) [inline spec/plugins/cors_spec.lua]: one of these functions throws an error when defined as `local` so I removed it.

**Comment by thefosk** [comment]
Merged and squashed with 6aebc40d6a833605ef99ffd1b444e989de49c528

**Pull request closed without merging**